tasks/winbuildscripts/Build-OmnibusTarget.ps1

```diff
@@ -71,11 +71,11 @@ Invoke-BuildScript `
     # Show the contents of the output package directories for debugging purposes
     Get-ChildItem -Path C:\omnibus-ruby\pkg\
     Get-ChildItem -Path "C:\opt\datadog-agent\bin\agent\"
-    Get-ChildItem -Path ".\omnibus\pkg\"
+    Get-ChildItem -Path ".\$env:OMNIBUS_PACKAGE_SUBDIR"

```

Comment:
would be nice to have these vars set to a default value so these scripts can work for local builds.
could put it in `common.ps1` maybe? might be good to throw an error if `CI` is set but these vars aren't, since we expect them to be set in the CI.